storage_service/locations/migrations/0007_file_add_new_fields.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'File.package'
+        db.add_column(u'locations_file', 'package',
+                      self.gf('django.db.models.fields.related.ForeignKey')(to=orm['locations.Package'], null=True),
+                      keep_default=False)
+
+        # Adding field 'File.source_package'
+        db.add_column(u'locations_file', 'source_package',
+                      self.gf('django.db.models.fields.TextField')(default='', blank=True),
+                      keep_default=False)
+
+        # Adding field 'File.accessionid'
+        db.add_column(u'locations_file', 'accessionid',
+                      self.gf('django.db.models.fields.TextField')(default='', blank=True),
+                      keep_default=False)
+
+        # Adding field 'File.origin'
+        db.add_column(u'locations_file', 'origin',
+                      self.gf('django.db.models.fields.CharField')(default='', unique=False, max_length=36, blank=True),
```

Comment:
I get a slightly different migration when I generate it. This column is not unique.  Perhaps this file needs to be re-generated?